P5/Source/Specs/lb.xml

```diff
@@ -89,15 +89,15 @@ soi-<lb break="no"/>xante &amp; dixhuytiesme livre.
       starts. The <att>n</att> attribute, if used, indicates the number or other value associated
       with the text between this point and the next <gi>lb</gi> element, typically the sequence
       number of the line within the page, or other appropriate unit. This element is intended to be
-      used for marking actual line breaks on a manuscript or printed page, at the point where they
+      used for marking line beginnings on a manuscript or printed page, at the point where they
       occur; it should not be used to tag structural units such as lines of verse (for which the
         <gi>l</gi> element is available) except in circumstances where structural units cannot
       otherwise be marked. </p>
     <p>The <att>type</att> attribute may be used to characterize the
-    line break in any respect. The more specialized attributes
+    line beginning in any respect. The more specialized attributes
     <att>break</att>, <att>ed</att>, or <att>edRef</att> should be preferred when the
     intent is to indicate whether or not the line
-    break is word-breaking, or to note the source from which it
+    beginning is word-breaking, or to note the source from which it
```

Comment:
Seems to me this one could just stay “break”, or perhaps “… whether or not the new topographic line implies a word break, or to note …”?

Reply:
I prefer the latter, personally, but can be swayed to revert to 'break' if that is the consensus. Thank you for your feedback.

Reply:
I find "line break" more natural in this specific context.